File: app/src/main/java/com/ellevsoft/utilmodule/MainActivity.java
package com.ellevsoft.utilmodule;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.ellevsoft.utility.UtilCommon;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        float aa = UtilCommon.convertDpToPixel(30, this);

    }
}

File: app/src/main/java/com/ellevsoft/utilmodule/MainActivity.kt
package com.ellevsoft.utilmodule

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity

import com.ellevsoft.utility.UtilCommon

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val pixel = UtilCommon.convertDpToPixel(30f, this)
        Log.d("test", "convert 30dp -> px: $pixel")
        val dp = UtilCommon.convertPixelsToDp(pixel, this)
        Log.d("test", "convert px -> dp: $dp")
    }
}
